matrix_compressor/matrix_compressor.h

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020-2021 PANDA GmbH
+
+#pragma once
+
+#include <blaze/Blaze.h>
+
+#include <tuple>
+#include <vector>
+
+namespace matrix_compressor {
+
+/* Compressed vector data */
+struct CompressedVector {
```

Comment:
I suggest to use ArchivedVectore and ArcivedMatrix, so that we avoid conflicts with names from blaze.